relay-server/src/actors/store.rs

```diff
@@ -522,17 +522,21 @@ enum KafkaMessage {
 
 impl KafkaMessage {
     /// Returns the partitioning key for this kafka message determining.
-    fn key(&self) -> &[u8] {
-        let event_id = match self {
-            Self::Event(message) => &message.event_id.0,
-            Self::Attachment(message) => &message.event_id.0,
-            Self::AttachmentChunk(message) => &message.event_id.0,
-            Self::UserReport(message) => &message.event_id.0,
-            Self::Session(message) => &message.session_id,
-            Self::Metric(_message) => return &[], // TODO(ja): Determine a partitioning key
+    fn key(&self) -> [u8; 16] {
+        let mut uuid = match self {
+            Self::Event(message) => message.event_id.0,
+            Self::Attachment(message) => message.event_id.0,
+            Self::AttachmentChunk(message) => message.event_id.0,
+            Self::UserReport(message) => message.event_id.0,
+            Self::Session(message) => message.session_id,
+            Self::Metric(_message) => Uuid::nil(), // TODO(ja): Determine a partitioning key
```

Comment:
nit: Could write `Uuid::new_v4()` here?

Reply:
Never mind, others could be `nil` as well.

Reply:
Yes, particularly session aggregates. I wanted to have just a single default, although we could also put it here.